client/public.go

```diff
@@ -110,6 +110,26 @@ type Profile struct {
 	AccessToken          string            `json:"-"`
 	Landing              string            `json:"landing" validation:"required"`
 	Program              string            `json:"program" validation:"required"`
+	Extensions           *[]*ExtensionData `json:"extensions,omitempty" sub-key:"-"`
```

Comment:
copied this "sub-key" from admin, don't know if it is needed or what

Reply:
It's the substitution key to use when generating a map for custom emails.  Definitely don't need it here...

Reply:
removed the subkey